src/clojush/pushgp/report.clj

```diff
@@ -41,6 +41,9 @@
       (println (name param) "=" (random/seed-to-string val))
       (println (name param) "=" val))))
 
+(defn print-genome [individual]
+  (pr-str (not-lazy (map #(dissoc % :uuid :parent-uuid) (:genome individual)))))
```

Comment:
I'm not sure if/how it would really manifest here, but this is about something different from my normal paranoia about laziness (which has to do with whether GC will properly reclaim storage for unrealized lazy sequences). The issue here relates specifically to printing. Sometimes, when you do some printing things, or converting to string things, with lazy sequences, you get a weird internal representation like "clojure.lang.LazySeq@c5d38b66", even if the sequence is fully evaluated. I do not fully understand what's going on here. It may be that using `pr-str` takes care of this, but I am not sure. To be safe, however, and because it is so deeply annoying to ever see something like "clojure.lang.LazySeq@c5d38b66" when you really wanted to see what's in the sequence, I have often taken to sprinkling `not-lazy` liberally through any code that prints strings from sequences.